org.dupot.easyflatpak.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.dupot.easyflatpak
+runtime: org.gnome.Platform
+runtime-version: "45"
```

Comment:
this runtime will be EOL in less than a couple of month.

Reply:
i change to freedesktop